static/app/components/orgRole.tsx

```diff
@@ -0,0 +1,123 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import ExternalLink from 'sentry/components/links/externalLink';
+import Link from 'sentry/components/links/link';
+import {Tooltip} from 'sentry/components/tooltip';
+import {IconInfo} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {Member, Organization} from 'sentry/types';
+import {getEffectiveOrgRole} from 'sentry/utils/orgRole';
+
+export const OrgRoleInfo = ({
+  organization,
+  member,
+}: {
+  member: Member;
+  organization: Organization;
+}) => {
+  const {orgRoleList} = organization;
+  const {orgRole, orgRolesFromTeams} = member;
+
+  const orgRoleFromMember = orgRoleList.find(r => r.id === orgRole);
+  if (!orgRoleFromMember) {
+    throw new Error();
```

Comment:
Rather than throwing an error, can we have the component return an error state instead?
will prevent the UI from breaking at least